### DISCUSSION TITLE
Provide safe list for language based playlist

### DISCUSSION THREAD

**Discussion by rrjanbiah** [iptv-org/community#500] [Ideas]
For example the category based playlist https://iptv-org.github.io/iptv/index.m3u has only 10064 channels where as the language based playlist https://iptv-org.github.io/iptv/index.language.m3u has 10530 channels. 

IOW, please provide safe list for https://iptv-org.github.io/iptv/index.language.m3u

**Comment by freearhey** [comment]
Could you give an example of such unsafe content from the https://iptv-org.github.io/iptv/index.language.m3u?

**Reply by rrjanbiah** [reply]
@freearhey I am sorry. I didn't closely check one by one. But the count difference suggests so

**Reply by freearhey** [reply]
the difference in number is due to the fact that some channels are broadcast in more than one language

**Reply by rrjanbiah** [reply]
Sorry. Closing as invalid

**Comment by rrjanbiah** [comment]
Sorry. Closing as invalid
